bpf/process/types/basic.h

```diff
@@ -1789,24 +1789,35 @@ struct {
 	__type(value, struct ratelimit_value);
 } ratelimit_map SEC(".maps");
 
+// The value has extra headroom to allow copying argument data without upsetting the verifier.
+// This is not hashed when the key is used in the ratelimit_map.
 struct {
 	__uint(type, BPF_MAP_TYPE_PERCPU_ARRAY);
 	__uint(max_entries, 1);
 	__type(key, __u32);
 	__type(value, __u8[sizeof(struct ratelimit_key) + 128]);
 } ratelimit_heap SEC(".maps");
 
+// This is zeroed memory that we NEVER write to, and use to copy over reusable heap in order
+// to zero it.
+struct {
+	__uint(type, BPF_MAP_TYPE_PERCPU_ARRAY);
+	__uint(max_entries, 1);
+	__type(key, __u32);
+	__type(value, __u8[sizeof(struct ratelimit_key) + 128]);
+} ratelimit_ro_heap SEC(".maps");
```

Comment:
would be nice to preface with tg_ratelimit_ro_heap and above so that bpftool reports them with tetragon 'tg' map preface. Not sure we have hit all the maps yet with the naming convention but will be nice going forward.